src/main/scala/com/cognite/sdk/scala/common/backends/BackpressureThrottleBackend.scala

```diff
@@ -0,0 +1,60 @@
+package com.cognite.sdk.scala.common.backends
```

Comment:
We should call it `com.cognite.sdk.scala.sttp` instead.

`GzipSttpBackend` should also be moved to that location.